packages/site/cms-site/src/iframebridge/IFrameBridge.tsx

```diff
@@ -127,7 +152,7 @@ export const IFrameBridgeProvider = ({ children }: PropsWithChildren) => {
 
             return newPreviewElementsData;
         });
-    }, [previewElements]);
+    }, [previewElements, childrenWrapperWidth]);
```

Comment:
Does this trigger an effect? I thought that refs never trigger effects. But maybe using a ref's field does 🤔

Reply:
Indirectly, the value is updated on re-render of the component, which then triggers the `useCallback` to update if the value changes.